packages/react-core/src/Authenticator/hooks/types.ts

```diff
@@ -143,6 +143,7 @@ export type SignUpBaseProps<FieldType = {}> = {
 
 export type VerifyUserProps<FieldType = {}> = {
   skipVerification: AuthenticatorMachineContext['skipVerification'];
+  unverifiedContactMethods: AuthenticatorMachineContext['unverifiedContactMethods'];
```

Comment:
`unverifiedContactMethods` is already passed to `VerifyUser` as its `fields` prop, so this is not required

Reply:
Thanks, removed!